python/cudf_polars/tests/conftest.py

```diff
@@ -8,3 +8,19 @@
 @pytest.fixture(params=[False, True], ids=["no_nulls", "nulls"], scope="session")
 def with_nulls(request):
     return request.param
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--executor",
+        action="store",
+        default="cudf",
+        choices=("cudf", "dask-experimental"),
```

Comment:
```suggestion
        choices=("pylibcudf", "dask-experimental"),
```

And matching throughout, since we don't actually use cudf anywhere.

Reply:
Done in https://github.com/rapidsai/cudf/pull/17262/commits/21e598af17a8755c09f8e14aa18b858830ba329e . Note that the downside is users who may want to specify the executor explicitly will have to type `"pylibcudf"` instead, which may be a bit confusing for those who don't work in cuDF's internals.